src/Styles/icon.scss

```diff
@@ -61,7 +64,7 @@
   &.CollapseRight:before {
     content: '\e94d';
   }
-  &.DoubleCaretVertical:before {
+  &.ChangeState:before {
```

Comment:
This will break multi-org, we use `DoubleCaretVertical` name in UI.

Reply:
FYI @wdoconnell , please correct me if I am misunderstanding.

Reply:
Uhh, it shouldn't have removed anything.

Reply:
Unless this is an icon not in our icomoon set.

Reply:
Correct, but if something was named `DoubleCaretVertical` i.e referred as `IconFont.DoubleCaretVertical`,

and now it's called ChangeState, Ui wont find it.

Reply:
Yup, looks like theres a mismach between naming. I'll update that in icomoon.

Reply:
Thank you.